kura/org.eclipse.kura.ble.provider/src/main/java/org/eclipse/kura/internal/ble/BluetoothLeDeviceImpl.java

```diff
@@ -39,7 +39,12 @@ public BluetoothLeDeviceImpl(tinyb.BluetoothDevice device) {
 
     @Override
     public BluetoothLeGattService findService(UUID uuid) throws KuraBluetoothResourceNotFoundException {
-        BluetoothGattService service = this.device.find(uuid.toString(), BluetoothLeDeviceImpl.TIMEOUT);
+        return findService(uuid, BluetoothLeDeviceImpl.TIMEOUT);
```

Comment:
You can use it directly without the static reference.

Reply:
I run the clean up and it keep it with the static reference.